docs/build.sh

```diff
@@ -21,8 +21,14 @@
 set -e
 rm -rf build 2> /dev/null
 rm -rf temp 2> /dev/null
+
+# copy source to temp dir
```

Comment:
There is also a `build.bat` file in this directory that might need to get updated

Reply:
Thanks, I had missed that.

Reply:
I don't have a convenient way to test on Windows, unfortunately